topics/chip-seq/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -0,0 +1,862 @@
+> > 2. As the cost of DNA sequencing falls, and new full genome sequencing technologies emerge, more genome sequences continue to be generated. Using these new sequences, new alignments are builts and the reference genomes improved (fewer gaps, fixed misrepresentations in the sequence, etc). The different reference genomes correspond to the different versions released (named build)
+> > 3. Wang et al. (2018) did ChIP-seq on mouse. So we should use the mouse reference genome. We will use mm10 (the latest build).
+> {: .solution }
+{: .question}
+
+> ### {% icon hands_on %} Hands-on: Mapping
+>
+> 1. **Bowtie2** {% icon tool %} with
+>    - *"Is this single or paired library"*: `Paired-end`
+>        - {% icon param-file %} *"FASTA/Q file #1"*: `trimmed reads pair 1` (output of **Trim Galore!** {% icon tool %})
+>        - {% icon param-file %} *"FASTA/Q file #2"*: `trimmed reads pair 2` (output of **Trim Galore!** {% icon tool %})
+>    - *"Will you select a reference genome from your history or use a built-in index?"*: `Use a built-in genome index`
+>        - *"Select reference genome"*: `Mouse (Mus musculus): mm10`
+>    - *"Save the bowtie2 mapping statistics to the history"*: `Yes`
+>
+> 2. Inspect the mapping stats
+>
+>    > ### {% icon question %} Questions
+>    >
+>    > How many reads where mapped? Uniquely or several times?
+>    >
+>    > > ### {% icon solution %} Solution
+>    > > The overall alignment rate is 98.64%. This score is quite high. If you have less than 70-80%, you should investigate the origin: contamination, etc.
+>    > >
+>    > > 43719 (90.27%) reads have been aligned concordantly exactly 1 time and 3340 (6.90%) aligned concordantly >1 times. The latter ones correspond to multiple mapped reads. Allowing for multiple  mapped reads increases the number of usable reads and the sensitivity of peak detection;
+however, the number of false positives may also increase. Here the number of uniquely mapped reads is sufficient to analyze to remove the multi-mapped reads.
+>    > {: .solution }
+>    {: .question}
+>
+{: .hands_on}
+
+The output of Bowtie2 is a BAM file (Binary Alignment Map), a compressed, binary file storing the sequences mapped to a reference sequence.
+
+## Inspection of a BAM file
+
+> ### {% icon hands_on %} Hands-on: Conversion into a SAM file
+>
+> 1. Inspect the {% icon param-file %} `aligned reads (BAM)` file (output of **Bowtie2** {% icon tool %})
+>
+{: .hands_on}
+
+A [BAM file](https://en.wikipedia.org/wiki/SAM_(file_format)) (or a SAM file, the non compressed version) consists:
+
+- A header section with the chromosome names and lengths (the lines starting with the `@` symbol)
```

Comment:
Is this really needed in a ChIP seq tutorial. The mapping one is imho a better place.